spec/controllers/v1.0/actions_controller_spec.rb

```diff
@@ -59,4 +58,40 @@
       end
     end
   end
+
+  describe 'POST /requests/:request_id/actions' do
+    context 'when request is actionable' do
+      let(:req) { create(:request, :with_context, :tenant_id => tenant.id) }
+      let!(:stage1) { create(:stage, :id => "1", :state => Stage::NOTIFIED_STATE, :request => req, :tenant_id => tenant.id) }
+      let!(:stage2) { create(:stage, :id => "2", :state => Stage::PENDING_STATE, :request => req, :tenant_id => tenant.id) }
+      let(:valid_attributes) { { :operation => 'cancel', :processed_by => 'abcd' } }
+
+      before do
+        allow(Group).to receive(:find)
```

Comment:
Would this `before` block work at the describe level so it doesn't have to be created at each context?